src/js/components/FormField/FormField.js

```diff
@@ -267,6 +268,7 @@ const FormField = forwardRef(
         }
         background={outerBackground}
         margin={abut ? abutMargin : margin || { ...formField.margin }}
+        round={border.position === 'outer' ? formField.round : undefined}
```

Comment:
A couple of thoughts:
1. Can we replace this current logic with `border.position === 'outer' && formField.round` ? (logic applies to line 210 as well)
2. In line 210 we are also applying `round` to the inner radius, wouldn't it look awkward? I'm having a hard time to imagine how it will look.

Reply:
Regarding 1. if we did that then we'd end up with `round={false}` when border.position is 'inside', but I think we'd rather have `round={undefined}`.

Regarding 2. line 210, we are setting the border and rounding there when border.position is 'inner'. Here, we are setting border and rounding when border.position is 'outer'.